Tests/PowerSkillTests/CustomEntitySearchTests/TestData.cs

```diff
@@ -606,14 +595,6 @@ public static void supportedTextandWordsTempInitializer()
                 });
             supportedMatchIndices.Add("German", new int[] { 387 });
             supportedConfidence.Add("German", new double[] { 0.5 });
-            supportedTextandWords.Add("Basic Overlap", new string[]
-                {
-                    @"a a a c a a c",
-                    @"a a",
-                    @"a a, a a, a a",
-                    @"a a"
-                });
-            supportedMatchIndices.Add("Basic Overlap", new int[] { 0, 2, 8 });
```

Comment:
So why are we removing tests? Are those cases no longer relevant?

Reply:
yes, i removed the code that lets you specify words in the request payload. You must use the config files now (similar to the schema we've defined for the 1st part integration)

---
In reply to: [330755115](https://github.com/Azure-Samples/azure-search-power-skills/pull/14#discussion_r330755115) [](ancestors = 330755115)